src/src/PluginStructs/P109_data_struct.cpp

```diff
@@ -85,6 +85,9 @@ bool P109_data_struct::plugin_init(struct EventStruct *event) {
   _varIndex        = event->BaseVarIndex;
   _relaypin        = P109_CONFIG_RELAYPIN;
   _relayInverted   = P109_GET_RELAY_INVERT;
+  _buttonInverted[0]   = P109_GET_BUTTON1_INVERT;
+  _buttonInverted[1]   = P109_GET_BUTTON2_INVERT;
+  _buttonInverted[2]   = P109_GET_BUTTON3_INVERT;
   _setpointTimeout = P109_CONFIG_SETPOINT_DELAY - P109_SETPOINT_OFFSET;
```

Comment:
Hm, you don't seem to have `Uncrustify` installed or enabled, for formatting the source code. that does a lot for readability.
[Docs on how to install & enable](https://espeasy.readthedocs.io/en/latest/Participate/PlatformIO.html#uncrustify)

Reply:
Formatted.